src/index.js

```diff
@@ -0,0 +1,64 @@
+import './style.css';
+// Create object with different hardcoded elements
+const todolist = [
+  {
+    index: 1,
+    description: 'Wash the dishes',
+    completed: false,
+  },
+
+  {
+    index: 1,
+    description: 'Complete todo list project',
+    completed: false,
+  },
+];
+class Tasks {
+  constructor() {
+    this.toDoList = [];
+  }
+
+  add(description, completed) {
+    this.toDoList = this.toDoList.concat({
+      index: Date.now(),
+      description,
+      completed,
+    });
+  }
+
+  remove(id) {
+    this.toDoList = this.toDoList.filter((list) => list.id !== Number(id));
+  }
+}
+// Create new tasks in the todolist
+const lists = new Tasks();
+const todoDiv = document.querySelector('.lists');
+const getTodoList = () => {
+  todolist.forEach((list) => {
```

Comment:
- [ ] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index.

**_Hint:_** Consider this code: `Array.sort((a, b) => a.index - b.index`